packit_service/service/api/webhooks.py

```diff
@@ -206,28 +208,94 @@ def post(self):
 
         return "Webhook accepted. We thank you, Gitlab.", HTTPStatus.ACCEPTED
 
-    @staticmethod
-    def validate_token():
+    def create_confidential_issue_with_token(self):
+        project_data = request.json["project"]
+
+        namespace, repo_name = project_data["path_with_namespace"].split("/")
+        http_url = project_data["git_http_url"]
+
+        project_token = ProjectTokenModel.get_project(
+            namespace=namespace, repo_name=repo_name, project_url=http_url
+        )
+
+        if not project_token:
```

Comment:
Here I am checking if the project token in present in the database or not. If present we can assume we have created an issue.

Reply:
Yes, I got it. My point is that saving tokens is now only an implementation detail how to get rid of the duplicates and we really don't care about the token value.